VenmoPayments/build.gradle

```diff
@@ -39,3 +38,10 @@ dependencies {
     androidTestImplementation deps.androidxJUnit
     androidTestImplementation deps.androidxEspressoCore
 }
+
+project.ext.name = "venmo-payments"
```

Comment:
Do the CI scripts for test / release already include running this module?

Reply:
Oh wow I'm not sure. We used to have to declare it explicitly in the build script 🤔

Reply:
Removed in: [29597c1](https://github.com/paypal/paypal-android/pull/267/commits/29597c1366264ab2ec3d66e787f8c927daef111a)